### ISSUE TITLE
"Getting Started": build-app gives "Failed to find esbuild"

### ISSUE THREAD

**Issue by erikv85** [purescript/spago#899]
Following the steps in the Getting Started guide gives the following output when reaching the section "Creating Executables":

```
$ spago build-app
[info] Build succeeded.
[error] Failed to find esbuild. See https://esbuild.github.io/getting-started/#install-esbuild for ways to install esbuild.
```

purs version 0.15.4 and spago 0.20.9, node 16.13.0 and npm 8.19.2, ubuntu 20 on x86.

Steps to reproduce:
```
mkdir my-project
cd my-project
spago init
spago install lists foldable-traversable

cat << EOF > src/Euler.purs
module Euler where

import Prelude

import Data.List (range, filter)
import Data.Foldable (sum)

ns = range 0 999

multiples = filter (\n -> mod n 3 == 0 || mod n 5 == 0) ns

answer = sum multiples
EOF

cat << EOF > src/Main.purs
module Main where

import Prelude

import Euler (answer)
import Effect.Console (log)

main = do
  log ("The answer is " <> show answer)
EOF

spago build
spago bundle-app
```

**Comment by ShaunApps** [comment]
Seems like esbuild needs to be installed globally. the following worked for me:

`npm install -g esbuild`

**Comment by thought2** [comment]
But isn't it quite common to have it installed locally? Would it maybe make sense to have a flag used like `--es-build-command 'yarn run esbuild'`?

**Comment by TernaryM01** [comment]
This is a defect in the "Getting Started" guide that should be fixed. Web development newbies like I am just trying out to learn PureScript can't figure out how to fix this. I have tried `npm install esbuild`, but `spago bundle-app` still gives the same error. I refuse installing esbuild globally until I get more clarity on this.

**Comment by mkanta** [comment]
I agree, esbuild shouldn't be installed globally. As a work-around, if you use npm run spago bundle-app with npx, ie npx spago bundle-app. This finds a local installation of esbuild. For other package managers, use the equivalent of npx.

**Comment by nicet** [comment]
if you use `npx spago bundle-app` directly, it will pop up `[error] Failed to fetch purs version.\n [error] Failed to run 'purs --version'`.
need edit the npm package.json:

```json
  "scripts": {
    "bundle": "spago bundle-app"
  },
```
and then the bundled `index.js` is produced.
